src/modules/ekf2/EKF/ekf.cpp

```diff
@@ -178,6 +180,16 @@ bool Ekf::update()
 	return false;
 }
 
+void Ekf::runRangeBeaconPositionEstimator(const imuSample sample) {
+
+    if (_pos_vel_estimator_beacon.isInitialised()) {
```

Comment:
Just give the data to the object and let it decide to use it or not. It knows if it's initialized or not.